src/test/java/ibm/jceplus/junit/base/BaseTestAES.java

```diff
@@ -103,35 +102,26 @@ public BaseTestAES(String providerName, int keySize) throws Exception {
         }
     }
 
-    //--------------------------------------------------------------------------
-    //
-    //
-    public void setUp() throws Exception {
-        byte[] encodedKey = new byte[(specifiedKeySize > 0 ? specifiedKeySize : 128) / 8];
+    @BeforeClass
+    public static void setUpBeforeClass() throws Exception {
         r.nextBytes(plainText512);
         r.nextBytes(plainText65536);
         r.nextBytes(plainText524288);
         r.nextBytes(plainText1048576);
         r.nextBytes(plainText16KB);
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        byte[] encodedKey = new byte[(specifiedKeySize > 0 ? specifiedKeySize : 128) / 8];
         r.nextBytes(encodedKey);
         key = new SecretKeySpec(encodedKey, 0, encodedKey.length, "AES");
     }
 
-    //--------------------------------------------------------------------------
-    //
-    //
-    public void tearDown() throws Exception {}
-
-    //--------------------------------------------------------------------------
-    //
-    //
     public void testAES() throws Exception {
         encryptDecrypt("AES");
     }
 
-    //--------------------------------------------------------------------------
-    //
-    //
     public void testAES_CBC_ISO10126Padding() throws Exception {
         try {
             cp = Cipher.getInstance("AES/CBC/ISO10126Padding", providerName);
```

Comment:
Is there any reason why we dont initialize a new cipher instance like line 290 such as  
`Cipher cp = Cipher.getInstance("AES/CBC/ISO10126Padding", providerName);`

Reply:
In those tests the `cp` variable is never used thus resulting in a checkstyle error. Also, since it's never used, it doesn't matter for multithreaded tests, so there is no need to have it local.